pkg/vm/engine/disttae/tools.go

```diff
@@ -533,30 +533,30 @@ func genColumnInfoExpr(accountId uint32, databaseId, tableId uint64) *plan.Expr
 		args = append(args, newColumnExpr(catalog.MO_COLUMNS_ATT_DATABASE_ID_IDX, types.T_varchar,
 			catalog.MoColumnsSchema[catalog.MO_COLUMNS_ATT_DATABASE_ID_IDX]))
 		args = append(args, newIntConstVal(databaseId))
-		left = plantool.MakeExpr("=", args)
+		left = plantool.MakeExpr(ctx, "=", args)
 	}
 	{
 		var args []*plan.Expr
 
 		args = append(args, newColumnExpr(catalog.MO_COLUMNS_ATT_RELNAME_ID_IDX, types.T_uint64,
 			catalog.MoTablesSchema[catalog.MO_COLUMNS_ATT_RELNAME_ID_IDX]))
 		args = append(args, newIntConstVal(tableId))
-		right = plantool.MakeExpr("=", args)
-		left = plantool.MakeExpr("and", []*plan.Expr{left, right})
+		right = plantool.MakeExpr(ctx, "=", args)
+		left = plantool.MakeExpr(ctx, "and", []*plan.Expr{left, right})
 	}
 	{
 		var args []*plan.Expr
 
 		args = append(args, newColumnExpr(catalog.MO_COLUMNS_ACCOUNT_ID_IDX, types.T_uint32,
 			catalog.MoTablesSchema[catalog.MO_COLUMNS_ACCOUNT_ID_IDX]))
 		args = append(args, newIntConstVal(accountId))
-		right = plantool.MakeExpr("=", args)
+		right = plantool.MakeExpr(ctx, "=", args)
 	}
-	return plantool.MakeExpr("and", []*plan.Expr{left, right})
+	return plantool.MakeExpr(ctx, "and", []*plan.Expr{left, right})
 }
 
 // genInsertExpr used to generate an expression to partition table data
-func genInsertExpr(defs []engine.TableDef, dnNum int) *plan.Expr {
+func genInsertExpr(ctx context.Context, defs []engine.TableDef, dnNum int) *plan.Expr {
```

Comment:
What is the purpose of passing ctx? I don't see any use for ctx.

Reply:
passing to `plantool.MakeExpr`